### DISCUSSION TITLE
Canvas text color

### DISCUSSION THREAD

**Discussion by White-Rabbit-Scientific** [beeware/toga#1671] [Ideas]
Objects such as lines, which can be applied to a `canvas` object, have a (pen) color property. `write_text` does not have this attribute. Any reasons for not adding this to the wish list?

**Comment by freakboy3742** [comment]
I'm not sure what you're referring to when you say drawing lines have a color property; `line_to()`, `arc()` etc don't accept a color attribute; you can set the color of a *stroke* and a *fill* - but that's used to set a context, which will be inherited by any draw command - including write_text. The `canvas` example demonstrates drawing text in different colors.
